Sources/Helpers/OAAppSettings.h

```diff
@@ -650,4 +650,6 @@ typedef NS_ENUM(NSInteger, EOARulerWidgetMode)
 @property (nonatomic) OAProfileBoolean *arrowsOnMap;
 @property (nonatomic) OAProfileBoolean *directionLines;
 
+@property (nonatomic) BOOL shouldShowWhatsNew;
```

Comment:
All bool properties are @property (nonatomic, assign) in this class & please add this property before the method declarations